backend/api/src/main/kotlin/software/shonk/application/service/ShorkService.kt

```diff
@@ -70,6 +69,22 @@ class ShorkService(private val shork: IShork) : ShorkUseCase {
         return Result.success(lobby.getStatus())
     }
 
+    override fun getAllLobbies(): List<LobbyStatus> {
+        return lobbies.keys.mapNotNull { lobbyId ->
+            getLobbyStatus(lobbyId).getOrNull()?.let { status ->
+                LobbyStatus(
+                    lobbyId = lobbyId,
+                    playersJoined =
+                        listOfNotNull(
+                            if (status.playerASubmitted) "playerA" else null,
+                            if (status.playerBSubmitted) "playerB" else null,
+                        ),
```

Comment:
This still has a bug. If a player has joined the lobby, but hasn't submitted any code yet, they wouldn't appear in the list of joined players.